.gitignore

```diff
@@ -1,5 +1,3 @@
-.gitignore
-_fs/*
```

Comment:
`_fs/$TARGET` is being created/written to as a part of building process. 

We should probably move target rootfs to the `_buid` subdir but IMHO for now you need to whitelist only `rootfs-skel` subdir:

```
!_fs/root-skel/
```